src/examples/sedov_blast_wave/update.cpp

```diff
@@ -0,0 +1,54 @@
+#include "update.hpp"
+
+const double max_dt_increase = 1.1;
```

Comment:
Could this be constexpr?

Reply:
Sure, I've just learnt that `const` and `constexpr` variables are not equivalent.
I changed all constants to `constexpr`.